src/renderer/util/fileSystem.js

```diff
@@ -33,6 +33,44 @@ export const getContentHash = content => {
   return getHash(content, 'utf8', 'sha1')
 }
 
+/**
+ * Check two image paths are identical
+ *
+ * @param {String} cwd The relative base path (project root or full folder path of opened file).
+ * @param {String} relativeName The relative directory name of image assets.
+ * @param {String} srcImagePath The source image file path.
+ * @returns {{identical: boolean, relPath: string}} Returns the idencial or not and relativePath of image file.
+ */
+export const IsIdenticalImagePath = async (cwd, relativeName, srcImagePath) => {
```

Comment:
I think you can use `isChildOfDirectory` from `common/filesystem` to keep code clean? The idea is to check whether the source image is inside the assets directory right?